platforms/hyperledger-fabric/configuration/roles/setup/fetch_config_block/Readme.md

```diff
@@ -0,0 +1,55 @@
+## ROLE: setup/fetch_config_block
```

Comment:
path should be setup/config_block/fetch and sign_update and like that for all config_block